week-3/quote-generator/quotes.js

```diff
@@ -491,3 +491,19 @@ const quotes = [
 ];
 
 // call pickFromArray with the quotes array to check you get a random quote
+
+const quote = pickFromArray(quotes);
+const p = document.querySelector('#quote');
+const author = document.querySelector('#author');
+p.innerHTML = `‟ ${quote.quote}`;
+author.innerHTML = `- ${quote.author}`;
+
+// new quote button
+
+const newQuote = document.querySelector('#new-quote');
+
+function changeQuote() {
```

Comment:
It looks like you have two copies of the contents of this function - one that you use the first time the page loads, and one for which you click the button. But they do the same thing.

Imagine you wanted to add one more step, e.g. adding a picture of the author. You'd have to add that code to two different places.

Can you think of a way to avoid repeating this code, so that you only have one copy of it?